protocol/messenger_contacts.go

```diff
@@ -18,21 +18,32 @@ import (
 	"github.com/status-im/status-go/protocol/transport"
 )
 
+const outgoingMutualStateEventSentDefaultText = "You sent a contact request to @%s"
```

Comment:
Oh, I didn't know these text lines are in status-go.
Shouldn't they be in UI (QML side) to have ability for cool things like
- text localisation
- interactive profile name (e.g. clicking opens profile popup)

?

Reply:
Ok, actually we're still able to detect the `contentType` and override the message text if we want to.

Reply:
This topic was discussed here: https://github.com/status-im/status-go/pull/3519#discussion_r1206799703. In general this text i's a fallback for clients that does not provide translations